tests/functional/include/tests/ft_utils.h

```diff
@@ -0,0 +1,21 @@
+// Leka - LekaOS
+// Copyright 2022 APF France handicap
+// SPDX-License-Identifier: Apache-2.0
+
+#pragma once
+
+#include "rtos/ThisThread.h"
+
+#include "boost/ut.hpp"
+
+using namespace std::chrono;
+using namespace boost::ut;
+
+void busy_sleep(uint64_t milliseconds)
```

Comment:
So in the end I think this function is not useful anymore.

Reply:
Great if it works! The function may not be useful today but we can always think about it in the future